internal/integration/api/volumes.go

```diff
@@ -71,7 +71,7 @@ func (suite *VolumesSuite) testDiscoveredVolumes(node string) {
 			Names: []string{"xfs"},
 		},
 		"EPHEMERAL": {
-			Names: []string{"xfs"},
+			Names: []string{"xfs", ""},
```

Comment:
:sweat_smile:

Reply:
I gave up. it doesn't affect anything in real life except for the test

Reply:
yeh, hopefully we find a good reproducer in future